rewrite-maven/src/main/java/org/openrewrite/maven/tree/ResolvedPom.java

```diff
@@ -307,6 +307,17 @@ public String getManagedVersion(String groupId, String artifactId, @Nullable Str
         return null;
     }
 
+    @Nullable
+    public String getManagedPluginVersion(String groupId, String artifactId) {
+        for (Plugin p : pluginManagement) {
+            if (p.matches(groupId, artifactId)) {
+                return getValue(p.getVersion());
+            }
+        }
+
+        return null;
+    }
+
```

Comment:
If we could pull out this and the change in `tree/Plugin.java` to the visitor that would help make the review and merge easier. Changes in the tree package require a coordinated release and deployment, and it seems we only use this once for now, so then having it near where it's used is preferred. We can always pull it up to the tree package later if we use this more often.

Reply:
There's some eager class loading involved that then requires a coordinate deployment. If we limit this change to not make those changes I don't have to involve others (yet).